website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,7 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## September 2024
 
+- **New**: Introducing the [microbatch incremental model strategy](/docs/build/incremental-microbatch) (beta), available in dbt Cloud Versionless and will soon be supported in dbt Core 1.9. The microbatch strategy allows for efficient, batch-based processing of large time-series datasets for improved performance and resiliency, especially when you're working with data that changes over time (like new records being added daily). To enable this feature in dbt Cloud, set `DBT_EXPERIMENTAL_MICROBATCH` to true in your project.
```

Comment:
@jtcohen6 @graciegoheen added a release note entry here

Reply:
Should we put this in October?

Reply:
Oh duh yes! Totally forgot